pkg/controller/deployment/deployment_controller.go

```diff
@@ -56,9 +59,14 @@ const (
 	MaxRetries = 5
 )
 
+func getDeploymentControllerKind() unversioned.GroupVersionKind {
```

Comment:
not sure i understand the reason for this. Are we saying its just used in one place and hence we should do this way. Keeping it consistent with getRSControllerKind() for now